app/src/main/java/com/cornellappdev/android/eatery/ui/viewmodels/ToggleViewModel.kt

```diff
@@ -0,0 +1,22 @@
+package com.cornellappdev.android.eatery.ui.viewmodels
+
+import androidx.lifecycle.ViewModel
+import com.cornellappdev.android.eatery.data.repositories.EateryRepository
+import com.cornellappdev.android.eatery.data.repositories.UserPreferencesRepository
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import javax.inject.Inject
+
+@HiltViewModel
+class ToggleViewModel @Inject constructor(
+) : ViewModel() {
+    //    default state is left side 'on'
+//    so _isToggled = false corresponds to left side 'on'
+    private val _isToggled = MutableStateFlow(false) // initial toggle state
+    val isToggled: StateFlow<Boolean> = _isToggled
+
+    fun toggle() {
+        _isToggled.value = !_isToggled.value
+    }
+}
```

Comment:
This seems a little unnecessary? All this is doing is basically communicating the toggle state of one boolean right?

I do believe that Zach took this out in his cleanup though. At least, I know he took it out of `FavoritesScreen.kt`. If after his merge goes in this file is not being used anymore, let's delete it.

Reply:
Makes sense, I moved the toggle functions to the FavoritesViewModel and deleted ToggleViewModel